rpc/transaction.go

```diff
@@ -179,21 +179,17 @@ const (
 	ResourceL2Gas
 )
 
-func (r Resource) MarshalJSON() ([]byte, error) {
```

Comment:
this is interesting case: `MarshalJSON()` returns incorrect values (they are not strings on byte level), but somehow it works (because of MarshalText() below I presume)